src/style.css

```diff
@@ -0,0 +1,162 @@
+body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  font-family: Roboto, sans-serif;
+}
+
+.card__informacion {
+  margin: 0 16px 16px;
+  width: 166px;
+}
+
+.card {
+  width: 198px;
+  height: 406px;
+  top: 1px;
+  left: 1px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+}
+
+.card__photo {
+  margin: 32px 19px 40px;
+  width: 160px;
+  height: 134px;
+  background-position: center;
+  background-size: 100% 100%;
+  background-image: url("./images/imac.jpag");
+  background-repeat: no-repeat;
+}
+
+.card__text {
+  margin-bottom: 4px;
+  color: #060b35;
+  width: 166px;
+  height: 36px;
+  top: 207px;
+  left: 17px;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  text-align: left;
+  text-underline-position: from-font;
+  text-decoration-skip-ink: none;
+}
+
+.card__code {
+  margin-bottom: 16px;
+  color: #616070;
+  width: 98px;
+  height: 14px;
+  top: 247px;
+  left: 17px;
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  text-align: left;
+  text-underline-position: from-font;
+  text-decoration-skip-ink: none;
+}
+
+.card__reviews {
+  display: flex;
+  margin-bottom: 24px;
+  justify-content: space-between;
+}
+
+.card__reviews-count {
+  color: #060b35;
+  width: 53px;
+  height: 16px;
+  display: flex;
+  justify-content: right;
+  align-items: end;
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  text-align: right;
+  text-underline-position: from-font;
+  text-decoration-skip-ink: none;
+}
+
+.card__price {
+  margin-bottom: 16px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.card__price-text {
+  color: #616070;
+  display: flex;
+  flex-direction: row;
+  width: 31px;
+  height: 18px;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  text-align: left;
+}
+
+.card__price-price {
+  color: #060b35;
+  justify-content: end;
+  width: 50px;
+  height: 18px;
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+  text-align: right;
+}
+
+.card__button-buy {
+  color: #fff;
+  background-color: #00acdc;
+  text-decoration: none;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
+  width: 166px;
+  height: 40px;
+  border-radius: 5px;
+  font-family: Roboto, sans-serif;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+}
+
+.card__button-buy[data-qa=hover]:hover {
+  border: 1px solid #00acdc;
+  color: #00acdc;
+  background-color: #fff;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars-star {
+  width: 16px;
+  height: 16px;
+  left: 80px;
+  border-radius: 0.5px;
+  margin-right: 4px;
+  background-image: url("./images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center center;
+}
+
+.stars-star:nth-child(-n+4) {
+  background-image: url("./images/star-active.svg");
+}
+
+.body {
```

Comment:
The `.body` class is defined but not used in the HTML file. Consider removing it if it's unnecessary, as the `body` element is already styled.